src/main/surfrdMod.F90

```diff
@@ -406,6 +407,27 @@ subroutine surfrd_get_data (begg, endg, ldomain, lfsurdat, actual_numcft)
 
     call check_sums_equal_1(wt_lunit, begg, 'wt_lunit', subname)
 
+    ! Select N dominant landunits
+    ! ---------------------------
+    ! n_dom_landunits set by user in namelist
+    ! Call resembles the surfrd_veg_all call to the same subr that selects
+    ! n_dom_pfts (also set by the user in the namelist)
+
+    call collapse_to_dominant(wt_lunit(begg:endg,:), istsoil, max_lunit, &
+                              begg, endg, n_dom_landunits)
+
+    ! UNDER CONSTRUCTION
```

Comment:
@slevisconsulting I'm confused about this commented out code here? Is this something coming later? If so can you comment more about it in terms of what and where?

Reply:
Oh, yes... That's the next thing I will work on. I made a placeholder for it while it was on my mind...

Reply:
Best to remove from this PR before merging?

Reply:
I think you should take it out, if you want this PR to go in to master without it.  If you are going to add it on your branch, and include it in the final PR that comes to master, then it's fine to have it in place. I'm somewhat OK with adding something as a placeholder for a future PR if it's going to happen soon. But, I'm not convinced this sort of thing really helps. So as a temporary thing on a branch, that goes away when it comes to master -- I'm totally fine with. As a temporary thing that does show up in master, I'm less OK with.

Reply:
Removed.